cli/internal/cmdutil/errors.go

```diff
@@ -0,0 +1,14 @@
+package cmdutil
```

Comment:
This package might be a good home for the code in `utilts/cmd.go`, especially if you end up using that for the `Help()` method.